lisa/tools/lscpu.py

```diff
@@ -26,6 +28,17 @@ def _windows_tool(cls) -> Optional[Type[Tool]]:
     def _check_exists(self) -> bool:
         return True
 
+    def get_architecture(self, force_run: bool = False) -> str:
+        result = self.run(force_run=force_run)
+        matched = self.__architecture.findall(result.stdout)
```

Comment:
The name `__architecture` is confusing with `_architecture`. So rename the `__architecture` like `__architecture_pattern`